Automattic/newspack-plugin
# Use grid to reorganise the reader revenue

### All Submissions:






### Changes proposed in this Pull Request:

This PR uses the freshly modified `Grid` component combined with `Card` to reorganise the Reader Revenue Wizard (except for the Address -- I will need to think about the margins and how I will want to redesign this screen)

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Check the different screens from the Reader Revenue

### Other information:
